rf-notebook/src/main/docker/docker-compose.yml

```diff
@@ -7,11 +7,12 @@ services:
       # jupyter notebook port
       - "8888:8888"
       # spark UI ports
-      - "4040:4040"
-      - "4041:4041"
-      - "4042:4042"
-      - "4043:4043"
-      - "4044:4044"
+      - "44040:4040"
```

Comment:
are these intended changes? i actually do have something like this on my local machine but not sure if others wanted that in there.